congame-doc/concepts.scrbl

```diff
@@ -0,0 +1,116 @@
+#lang scribble/manual
+
+@(require [for-label racket/base]
+          "doc-util.rkt"
+          scribble/bnf)
+
+@title{Overview}
+
+What follows is a high-level overview of Congame's systems and how they all work together. If you've
+just finished the @seclink["intro"]{introductory tour} then this will give you some more context for
+what you saw.
+
+@section{Programming in Racket}
+
+The Congame system is built entirely on the Racket programming language, and the Congame studies 
+you will create are essentially Racket programs.
+
+If you only read these docs, you should be able to pick up enough Racket to write studies. But
+Congame gives you access to lots of Racket capabilities. If you take a few minutes to learn about
+Racket programming in general, you will have a better understanding of what’s happening in your
+study’s code and how to fix it when it isn’t working.
+
+Here are some great resources for someone new to Racket:
+
+@itemlist[
+
+@item{@secref["intro" #:doc '(lib "scribblings/guide/guide.scrbl")] and @secref["to-scheme" #:doc
+'(lib "scribblings/guide/guide.scrbl")], the first two sections of @other-doc['(lib
+"scribblings/guide/guide.scrbl")].}
+
+@item{@other-doc['(lib "scribblings/quick/quick.scrbl")]}
+
+]
+
+If you are already even a little handy with other programming languages, you may want to bookmark
+the @secref["data" #:doc '(lib "scribblings/reference/reference.scrbl")] chapter of @italic{The
+Racket Reference}, to build up your vocabulary of the most common functions.
+
+
+@section{Basic Congame Concepts}
+
+A @deftech{step} is a point within a @tech{study} at which we provide information to
+a participant, and/or collect a response from them. Steps in Congame correspond to
+individual web pages.
+
+A @deftech{study} is a series of @tech{steps} and a transition graph that controls how
+study participants proceed through those steps.
+
+A @deftech{study instance} is a discrete time when a @tech{study} is run on a particular
+server. A study instance has a beginning and an end date/time. Congame keeps separate sets of
+results for each study instance.
+
+A @deftech{replication} is a duplicate of a @tech{study instance}, created specifically to test
+whether a study instance's results can be replicated — or, to run the study with new participants,
+or simply to keep a copy of the data.
+
+@section{Building Studies}
+
+Two ways to author studies, depending on whether you're running your own Congame server.
```

Comment:
I have been doing this with the `@mark` function I added, which highlights text a dull yellow. There are a lot more instances where I would mark up this kind of text, but I also don’t want the result to be distracting or confusing for your readers.

What if, instead of using `@mark` for this purpose, I add a `@tktk` function (named from an [editing convention](https://en.wikipedia.org/wiki/To_come_(publishing))) that made the text light grey, surrounded it in brackets, and put some kind of small note in the margin?

Reply:
Yes, that's perfect.